src/commands/support/navdata.ts

```diff
@@ -2,19 +2,24 @@ import { CommandDefinition } from '../../lib/command';
 import { CommandCategory } from '../../constants';
 import { makeEmbed, makeLines } from '../../lib/embed';
 
-const NAVDATA_URL = 'https://cdn.discordapp.com/attachments/785976111875751956/912394143844696154/unknown.png';
-
 export const navdata: CommandDefinition = {
-    name: 'navdata',
-    description: 'Provides help with Navigraph navdata reinstall',
+    name: ['navdata'],
```

Comment:
Do we need the `[ ]` on commands without an alias? I have seen some files with and without. Is it correct to have all with? If so, I feel a refactor incoming haha

Reply:
I want to make it uniform and a standard. We can make this part of the split command effort as as we will be touching so many command anyway?

Reply:
> Do we need the `[ ]` on commands without an alias? I have seen some files with and without. Is it correct to have all with? If so, I feel a refactor incoming haha

No it is not needed. Both will work fine. It looks cleaner without the brackets on the single element though.

Reply:
I personally prefer to have a consistent standard, which would be with `[ ]`. The advantage of it, is that it automatically becomes clear that multiple commands/aliases can be added, even when looking at existing simple commands and the code looks the same everywhere.

 If there's general disagreement, I'll remove it.

Reply:
Dont see the need, both work just as fine.

Reply:
Sounds good, will refactor for uniformity at some point